Tasks/a2_priority_queue.py

```diff
@@ -1,43 +1,50 @@
-"""
-Priority Queue
-
-Queue priorities are from 0 to 5
-"""
-
-
-def enqueue(elem, priority: int = 0) -> None:
-	"""
-	Operation that add element to the end of the queue
-
-	:param elem: element to be added
-	:return: Nothing
-	"""
-	return None
-
-
-def dequeue():
-	"""
-	Return element from the beginning of the queue
-
-	:return: dequeued element
-	"""
-	return None
-
-
-def peek(ind: int = 0, priority: int = 0):
-	"""
-	Allow you to see at the element in the queue without dequeuing it
-
-	:param ind: index of element (count from the beginning)
-	:return: peeked element
-	"""
-	return None
-
-
-def clear() -> None:
-	"""
-	Clear my queue
-
-	:return: None
-	"""
-	return None
+"""
+Priority Queue
+Queue priorities are from 0 to 5
+"""
+
+priority_queue = {i: [] for i in range(6)}
+
+
+def enqueue(elem, priority: int = 0) -> None:
+	"""
+	Operation that add element to the end of the queue
+	:param elem: element to be added
+	:return: Nothing
+	"""
+	global priorrity_queue
+	print(priority_queue[priority])
+	priority_queue[priority].append(elem)
+	return None
+
+
+def dequeue():
+	"""
+	Return element from the beginning of the queue
+	:return: dequeued element
+	"""
+	global priority_queue
+	for i in range(6):
+		if priority_queue[i]:
+			return priority_queue[i].pop(0)
+
+
+def peek(ind: int = 0, priority: int = 0):
+	"""
+	Allow you to see at the element in the queue without dequeuing it
+	:param ind: index of element (count from the beginning)
+	:return: peeked element
+	"""
+	global priority_queue
+	return priority_queue[priority][ind]

```

Comment:
По хорошему, дефолтные значения должны показывать элементы, даже если в 0-вом приоритете их нет, чтобы пользователь мог видеть существующие, а не перебирать все.
Я бы сделал что-то вроде 
```suggestion
	return list(itertools.chain.from_iterable(a.values()))[ind]
```
т.е. слепил все очереди в одну и вывел ind с начала (чтобы если были элементы только с 5-м приоритетом, все равно показало бы)
Неблокирующий